docs/.spelling

```diff
@@ -0,0 +1,143 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+liveness
+pre-shared
+Django
+Blockchain-based
+Asynchronisation
+asynchronisation
+_i_
+_j_
+Wooldridge
+driverless
+AEA
+AEAs
+Fetch.ai
+blockchains
+blockchain
+ACAs
+aries
+Alice_AEA
+Faber_AEA
+Alice_ACA
+Faber_ACA
+webhook
+webhooks
+localhost
+aca-py
+testnet
+config
+multiaddress
+my_new_aea
+yaml
+regex
+scaffolded
+Ethereum
+stdlib
+VSCode
+DecisionMaker
+booleans
+_average_rating_
+_ebook_available_
+Colab
+Dockerfile
+SDKs
+TaskManager
+Teardown
+trustless
+deployer
+Ganache
+Ropsten
+faucet
+Fetch.ai.
+unregisters
+teardown
+unregister
+seller_AEA
+buyer_AEA
+Agentland
+generic_seller
+instantiable
+startup
+backend
```

Comment:
try `back-end` or `back end`.
The last one seems preferable, although less widespread.
https://en.wikipedia.org/wiki/Front_end_and_back_end

Reply:
its a valid noun, search `define: startup`